taskvine/src/manager/vine_manager.c

```diff
@@ -2642,7 +2642,7 @@ static struct vine_task *vine_manager_sched_worker_transfer(struct vine_manager
 					if((remote_info = hash_table_lookup(peer->current_files, f->cached_name)) && remote_info->in_cache)
 					{
 						char *peer_source =  string_format("worker://%s:%d/%s", peer->transfer_addr, peer->transfer_port, f->cached_name);
-						if(vine_current_transfers_source_in_use(q, peer_source) < VINE_FILE_SOURCE_MAX_TRANSFERS)
+						if(vine_current_transfers_source_in_use(q, peer_source) < VINE_WORKER_SOURCE_MAX_TRANSFERS)
```

Comment:
Use file_source_max_transfers here?

Reply:
For now, workers still have only 1 transfer port open. So they may have only one connection.